network/dns_frame.ksy

```diff
@@ -0,0 +1,121 @@
+meta:
+  id: dns_packet
+  title: DNS (No support for Auth-Name + Add-Name for simplicity)
+  application: https://www.ietf.org/rfc/rfc1035.txt
+  endian: be
+seq:
+  - id: transaction_id
+    doc: "ID to keep track of request/responces"
+    type: u2
+  - id: flags
+    type: u2
+    doc: "Please add support for bit-flags!"
+    enum: flags_type
+  - id: questions
+    doc: "How many questions are there"
+    type: u2
+  - id: answer_rrs
+    doc: "How many answers are there"
+    type: u2
+  - id: authority_rrs
+    doc: "How many authority answers are there"
+    type: u2
+  - id: additional_rrs
+    doc: "How many additional answers are there"
+    type: u2
+  - id: queries
+    type: query
+    repeat: expr
+    repeat-expr: questions
+  - id: answers
+    type: answer
+    repeat: expr
+    repeat-expr: answer_rrs
+types:
+  query:
+    seq:
+      - id: name
+        type: domain_name
+      - id: type
+        type: u2
+        enum: type_type
+      - id: class
+        type: u2
+        enum: class_type
+  answer:
+    seq:
+      - id: name
+        type: domain_name
+      - id: type
+        type: u2
+        enum: type_type
+      - id: class
+        type: u2
+        enum: class_type
+      - id: ttl
+        doc: "Time to live"
```

Comment:
Um, adding unit of measurement (i.e. that it's specified in seconds) would be good for documentation ;) Even RFC 1035 specifies that.